mkdocs.yml

```diff
@@ -25,7 +25,8 @@ nav:
   # - lhc-sm.md
   - beams_dynamics.md
   - machine_learning.md
+  - awake.md
+  # - outreach.md
   - apache_spark.md
+  - datatools.md
   - deeplearning-GPU.md
-  - outreach.md
```

Comment:
Outreach can stay at the end too? And we can have awake further down if you want.